### ISSUE TITLE
CPU use 100% after enable cpptools

### ISSUE THREAD

**Issue by BigVan** [microsoft/vscode-cpptools#1281]
- VSCode Version:
1.8.1

- OS Version:
CentOS 7.3

Steps to Reproduce:

1. yum install code
2. just open a cpp file and edit it.

this 'top' shows:
```
 PID    USER    PR  NI    VIRT    RES     SHR S  %CPU %MEM     TIME+ COMMAND                                                                                                                                                           
 58858 bigvan    20   0 2775708 444304  58500 R 106.7    2.7   2:28.35 code                                                                                                                                                              
 58978 bigvan    20   0 1191328 222404  20832 S  60.0    1.4   1:33.63 code 
```

Reproduces without extensions: No

extensions:
C/C++ 0.14.2
Go 0.6.67

It returns to normal after I disable c/c++ extension

**Comment by bobbrow** [comment]
The language server doesn't work on CentOS 7 yet.  We will be testing some fixes next week to remove the GLIBC 2.18 dependency so that the extension will run on CentOS and related distros.  There is a temporary workaround you can try in issue #19.

**Comment by sean-mcmanus** [comment]
Fixed with 0.14.4.
